client/components/utils/IncDrink.js

```diff
@@ -0,0 +1,8 @@
+export const incDrink = async (id) => {
+  try {
+    //check if user token exists, if not, redirect to login/register page. otherwise continue:
+    //check if active tab already exists. if not, create new tab and new order and add on drink. if so, add onto existing order
```

Comment:
@justinmattos I think here is where we could add the put request that you mentioned

Reply:
Indeed, this would be a good place for it